node/runtime/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
 	spec_version: 92,
-	impl_version: 93,
+	impl_version: 94,
```

Comment:
This change is not required. Just ignore the `check-runtime` pipeline

Reply:
Then it's pointless imho, can't we just make the CI report a warning instead of an error? Or add a label without breaking the build?

No one is going to look inside the PR to see that it is actually fine that it's red, I think it might introduce a really bad habit of treating red CI as something normal.

There is either a green mark next to the PR or the PR is broken, we shouldn't introduce anything in between.

Reply:
@tomusdrw I agree.

Reply:
THe problem is that Github does not support it. But people should just look at the pipelines, just the crucial ones are marked as required. If something fails that is not required, the reviewing person should be aware and be able to help the pr author.